scipy/stats/_qmc.py

```diff
@@ -883,6 +977,177 @@ def reset(self):
         return self
 
 
+class OptimalDesign(QMCEngine):
+    """Optimal design.
+
+    Optimize the design by doing random permutations to lower the centered
+    discrepancy [1]_.
+
+    The specified optimization `method` is used to select a new set of
+    permutations to perform. If `method` is None, *basinhopping* optimization
+    is used. `niter` set of permutations are performed.
+
+    Centered discrepancy-based design shows better space filling robustness
+    toward 2D and 3D subprojections. Distance-based design shows better space
+    filling but less robustness to subprojections [2]_.
+
+    Parameters
+    ----------
+    d : int
+        Dimension of the parameter space.
+    start_design : array_like (n, d), optional
+        Initial design of experiment to optimize. `LatinHypercube`
+        is used to generate a first design otherwise.
```

Comment:
If a `start_design` is provided, all future calls to `random` will start from it, so they are not very distinct. This may be confusing.

Reply:
Indeed. I guess I will then just restart from a blank LHS?